compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -581,6 +586,19 @@ void addMLIRAIRAIELoweringPasses(OpPassManager &passManager, bool packPeel) {
 
   passManager.addPass(xilinx::airrt::createAIRRtToNpuPass());
   passManager.addPass(createCanonicalizerPass());
+
+  // Now lower using the AIE passes used by the mlir-air/mlir-aie flow.
+  passManager.addPass(createLowerAffinePass());
+  OpPassManager &devicePM = passManager.nest<xilinx::AIE::DeviceOp>();
```

Comment:
Can the other aie2xclbin flags also be removed now? aie2xcbin-print-ir-before-all, etc.

Reply:
I thought about this, but no, only the context is shared (which is what the threading is lin ked to). The other passes use a new pass manager so the flags have to be set separately for them. Of course there might be a way to parse the top level flags but thats something to think on seperately.

Reply:
Makes sense, future work.